spec/components/collections/work_row_component_spec.rb

```diff
@@ -0,0 +1,16 @@
+# typed: false
```

Comment:
Do we want this blank test?

Reply:
We already have many such blank component tests, unfortunately, so this is at least consistent with current practice.